Packs/QRadar/Integrations/QRadar_v3/QRadar_v3.yml

```diff
@@ -2668,11 +2691,58 @@ script:
     - contextPath: QRadar.LocalDestinationIP.SourceAddressIDs
       description: List of source address IDs associated with the destination address.
       type: Unknown
+  - arguments:
```

Comment:
Continuing my previous comment please make sure to update the descriptions here as well.